### PR TITLE
Nits for protocol discovery

### PR DESCRIPTION
**Pull request by Hixie**
Rather than complain about these in a code review I figured I'd just fix them. :-)

cc @tvolkert

### CODE REVIEW

File: packages/flutter_tools/lib/src/protocol_discovery.dart [hunk 1]
```diff
@@ -9,71 +9,83 @@ import 'base/port_scanner.dart';
...
+      _completer.completeError(new ToolExit('Timeout while attempting to retrieve URL for $serviceName'));
+    });
+  }
+
+  factory ProtocolDiscovery.observatory(
+    DeviceLogReader logReader, {
+    DevicePortForwarder portForwarder,
+    int hostPort,
+  }) {
+    const String _kObservatoryService = 'Observatory';
```
Comment (tvolkert) [inline packages/flutter_tools/lib/src/protocol_discovery.dart]: No need for the underscore anymore (or below in the `diagnosticService` factory constructor)

**Review by tvolkert — APPROVED** [review]
LGTM

File: packages/flutter_tools/lib/src/protocol_discovery.dart (hunk 1 above)
Comment (Hixie) [inline packages/flutter_tools/lib/src/protocol_discovery.dart]: Fixed.

**Comment by Hixie** [comment]
Will land on green.

**Pull request merged**